.github/workflows/ci.yml

```diff
@@ -213,11 +213,11 @@ jobs:
       matrix:
         include:
           - flavor: asan
-            cc: clang-13
+            cc: clang
```

Comment:
are we getting something newer like 14/15? seems like the GHA images are updated at least once per year, so if clang is fairly recent then yes we don't need the https://apt.llvm.org/llvm.sh script.

Reply:
Clang 14